docs/launchpad/guides/arbitrum-archive-kubernetes-guide.md

```diff
@@ -2,112 +2,117 @@
 ---
 # Arbitrum Archive Mainnet Node Guide
 
-:::warning
-This Quick Start guide has not yet been updated for Launchpad V2.
-:::
+## Introduction
+This guide provides an end-to-end walkthrough for setting up an Indexer on the Graph Protocol Mainnet for the Arbitrum One network. It details the steps for deploying both Arbitrum Classic and Arbitrum Nitro.
 
-This guide is intended to be an end to end walk-through of running an Arbitrum Archive Mainnet Node in an existing Kubernetes cluster.
-Sync times are reported to be in the range of 1 week on dedicated hardware. The node consists of 2 parts, the classic part and the nitro hardfork. The classic part is only required to request archive data for blocks before the hardfork and takes the aforementioned 1 weeks to sync from scratch. The nitro history is shorter and can be quickly synced within 3 days.
+## Sync Duration
+- **Arbitrum Classic**: Approximately 1 week is required to sync on dedicated hardware.
+- **Arbitrum Nitro**: Typically syncs within 3 days due to a shorter historical data span.
```

Comment:
pretty sure this is no longer the case (the 3 days). Maybe we want to drop these so specific times? which not only are "hardware dependent" to a degree, also are hard to keep up to date - would need being rectified every few months